taco-team-build.js

```diff
@@ -201,6 +266,9 @@ function buildProject(cordovaPlatforms, args, /* optional */ projectPath) {
             promise = promise.then(function () {
                 // Build app with platform specific args if specified
                 var callArgs = utilities.getCallArgs(platform, args, cordovaVersion);
+                ensureProjectXcode8Compatibility(projectPath, platform, callArgs);
```

Comment:
This promise should be `return`'d to make sure that it resolves before continuing the next `then`